Find a functionally equivalent code:android.media.MediaRouter.removeUserRoute:COMMENT
Method Modifier: public      
Comment:/**
 * Remove an app-specified route for media from the MediaRouter.
 *
 * @param info Definition of the route to remove
 * @see #addUserRoute(UserRouteInfo)
 */

@@ -1,3 +1,3 @@
 {
-    removeRoute(info);
+    removeRouteStatic(info);
 }

